guardian/cdk-cli
# feat: Add the CfnInclude construct to the generated code

## What does this change?



Adds a new optional flag to the `new` command for the location of a YAML template file
and adds a `CfnInclude` construct to the file.

## How to test



- Create a YAML CFN template (e.g. `cfn.yaml`)
- Run `./bin/run new --app amigo --stack deploy --yaml-template-location cfn.yaml`
- Open the generated project
- Install dependencies (`yarn install`)
- Run the linter (`yarn lint --fix`) - it's quite difficult to produce code that passes linting, rather than struggle, we can just ask users to run `lint --fix` and move on. We'll eventually run this automatically.
- Synth the stack (`yarn synth`)
- Observe a successfully synthed stack that!

## How can we measure success?



MORE GENERATION!
